packages/azure-kusto-data/src/cloudSettings.ts

```diff
@@ -64,7 +64,7 @@ class CloudSettings {
         } catch (ex) {
             if (axios.isAxiosError(ex)) {
                 // Axios library has a bug in browser, not propagating the status code, see: https://github.com/axios/axios/issues/5330
-                if ((ex.response?.status === 404 && isNode) || (ex.code === axios.AxiosError.ERR_NETWORK && !isNode)) {
+                if ((ex.response?.status === 404 && isNode) || (!isNode && (!ex.code || ex.code === "ERR_NETWORK" ))) {
```

Comment:
1. The isNode should be in the same place in both conditions (so if you moved it to the front on the right, it should also be in the left)
2. Why did you change it from a constant to a string literal?
3. What's the change here, functionally?